docs/querying/basics.md

```diff
@@ -95,8 +95,9 @@ For example, this selects all `http_requests_total` time series for `staging`,
 
     http_requests_total{environment=~"staging|testing|development",method!="GET"}
 
-Label matchers that match empty label values also select all time series that do
-not have the specific label set at all. Regex-matches are fully anchored.
+Label matchers that match empty label values also select all time series that
+do not have the specific label set at all. Regex-matches are fully anchored. It
+is possible to have multiple matchers for the same label name.
```

Comment:
This still doesn't clarify that they are AND-ed together vs. OR-ed. I also think the AND property should be explicitly stated somewhere for matchers in general, not just for two matchers for the same label name.

Reply:
That's already covered a few paragraphs up.

Reply:
I was thinking of stating it more explicitly rather than just as an example. But that would perhaps imply making this whole document look more like a formal reference rather than a casual explanation. Maybe a task for another day...

:+1: then